### PR TITLE
User Guide update - 3D Styling for vector data

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add 3D Styling for vector data on Layer Settings section

*Documentation section involved*
- [x] User Guide
- [ ] Developer Guide

## Other useful information
Read the Docs file [here](https://elena-mapstore-docs.readthedocs.io/en/8414_userguideu/user-guide/layer-settings/#styling-on-the-3d-navigation)

### CODE REVIEW

File: docs/user-guide/layer-settings.md [hunk 1]
```diff
@@ -276,6 +280,33 @@ The [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) **Visual Styl
...
+    
+    * For the *Line symbolizers*: the *Line cap* and *Line join* options are not available as properties in Cesium
+
+Furthermore, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) adds some customization options, for **WFS layers**, in the *Cesium Style Editor* which are:
+
+* **Bring to front** to bring the Mark or the Icon in front of the **3d Tiles** layer (This option is available for Icon and Mark symbolizers).
+
+<img src="../img/layer-settings/bring-to-front.jpg" class="ms-docimage">
+
+* **Polygon type** to choose whether the classification should affect `3D`, `Terrain` or `Both`. (This option is available for File symbolizers)
```
Comment (allyoucanmap) [inline docs/user-guide/layer-settings.md:301]:
```suggestion
* **Polygon type** to choose whether the classification, drape effect, should affect `3D`, `Terrain` or `Both`. (This option is available for Fill symbolizers)
```

File: docs/user-guide/layer-settings.md [hunk 2]
```diff
@@ -276,6 +280,33 @@ The [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) **Visual Styl
...
+
+<img src="../img/layer-settings/polygon-type.jpg" class="ms-docimage">
+
+* **Clap to ground** to enable/disable the boolean Property specifying whether the polyline should be clamped to the ground (This option is available for Line and File symbolizers).
```
Comment (allyoucanmap) [inline docs/user-guide/layer-settings.md:305]:
```suggestion
* **Clap to ground** to enable/disable the boolean Property specifying whether the polyline should be clamped to the ground (This option is available for Line and Fill symbolizers).
```

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

**Review by allyoucanmap — APPROVED** [review]

**Comment by allyoucanmap** [comment]
@ElenaGallo please test in dev if everything has been deployed as expected

**Comment by tdipisa** [comment]
@ElenaGallo please backport to 2022.02.xx. Thank you.

**Pull request merged**